element_array_ephys/ephys_acute.py

```diff
@@ -504,6 +577,15 @@ class ClusterQualityLabel(dj.Lookup):
 
 @schema
 class ClusteringTask(dj.Manual):
+    """A clustering task to spike sort electrophysiology datasets.
+
+    Attributes:
+        EphysRecording (foreign key): EphysRecording primary key.
+        ClusteringParamSet (foreign key): ClusteringParamSet primary key.
+        clustering_outdir_dir (varchar (255) ): Relative path to output clustering results. 
+        task_mode (enum('load', 'trigger') ): Either loads or computes clustering.
```

Comment:
Let's just label this as (enum) and then give the `options` in the description. See comment in Probe